index.html

```diff
@@ -500,8 +500,8 @@
         <dfn><a href=
         "https://heycam.github.io/webidl/#idl-ArrayBuffer">ArrayBuffer</a></dfn>,
         <dfn><a href=
-        "https://heycam.github.io/webidl/#idl-ArrayBufferView">ArrayBufferView</a></dfn>,
-        and the following exception names are defined in [[!WEBIDL-1]]:
+        "https://heycam.github.io/webidl/#ArrayBufferView">ArrayBufferView</a></dfn>,
+        and the following exception names are defined in [[!WEBIDL-2]]:
```

Comment:
LGTM

Okay, on reading the full spec this list is separate from the "interface definitions" statement above, so deserves its own cross reference.